pkg/trace/api/api_test.go

```diff
@@ -586,6 +588,253 @@ func TestHandleStats(t *testing.T) {
+			RemoveQueryString: true,
+			RemovePathDigits:  true,
+		},
+		RemoveStackTraces: false,
+		Redis:             config.Enablable{Enabled: true},
+		Memcached:         config.Enablable{Enabled: false},
+	}
+	rcv := newTestReceiverFromConfig(&config.AgentConfig{
+		Enabled:    true,
+		Hostname:   "test.host.name",
+		DefaultEnv: "prod",
+		ConfigPath: "/path/to/config",
+		Endpoints: []*config.Endpoint{{
+			APIKey:  "123",
+			Host:    "https://target-intake.datadoghq.com",
+			NoProxy: true,
+		}},
+		BucketInterval:   time.Second,
+		ExtraAggregators: []string{"agg:val"},
+		ExtraSampleRate:  2.4,
+		TargetTPS:        11,
+		MaxEPS:           12,
+		ReceiverHost:     "localhost",
+		ReceiverPort:     8111,
+		ReceiverSocket:   "/sock/path",
+		ConnectionLimit:  12,
+		ReceiverTimeout:  100,
+		MaxRequestBytes:  123,
+		StatsWriter: &config.WriterConfig{
+			ConnectionLimit:    20,
+			QueueSize:          12,
+			FlushPeriodSeconds: 14.4,
+		},
+		TraceWriter: &config.WriterConfig{
+			ConnectionLimit:    21,
+			QueueSize:          13,
+			FlushPeriodSeconds: 15.4,
+		},
+		StatsdHost:                  "stastd.localhost",
+		StatsdPort:                  123,
+		LogLevel:                    "WARN",
+		LogFilePath:                 "/path/to/logfile",
+		LogThrottling:               false,
+		MaxMemory:                   1000000,
+		MaxCPU:                      12345,
+		WatchdogInterval:            time.Minute,
+		ProxyURL:                    u,
+		SkipSSLValidation:           false,
+		Ignore:                      map[string][]string{"K": {"1", "2"}},
+		ReplaceTags:                 []*config.ReplaceRule{{Name: "a", Pattern: "*", Repl: "b"}},
+		AnalyzedRateByServiceLegacy: map[string]float64{"X": 1.2},
+		AnalyzedSpansByService:      map[string]map[string]float64{"X": {"Y": 2.4}},
+		DDAgentBin:                  "/path/to/core/agent",
+		Obfuscation:                 obfCfg,
+	})
+	defer func(old string) { info.Version = old }(info.Version)
+	defer func(old string) { info.GitCommit = old }(info.GitCommit)
+	defer func(old string) { info.BuildDate = old }(info.BuildDate)
+	ff, ok := os.LookupEnv("DD_APM_FEATURES")
+	if !ok {
+		defer os.Unsetenv("DD_APM_FEATURES")
+	} else {
+		defer func(old string) { os.Setenv("DD_APM_FEATURES", old) }(ff)
+	}
+	os.Setenv("DD_APM_FEATURES", "feature_flag")
+	info.Version = "0.99.0"
+	info.GitCommit = "fab047e10"
+	info.BuildDate = "2020-12-04 15:57:06.74187 +0200 EET m=+0.029001792"
+	h := rcv.makeInfoHandler()
+	rec := httptest.NewRecorder()
+	req := httptest.NewRequest("GET", "/info", nil)
+	h.ServeHTTP(rec, req)
+	if rec.Body.String() != `{
+	"Version": "0.99.0",
+	"GitCommit": "fab047e10",
+	"BuildDate": "2020-12-04 15:57:06.74187 +0200 EET m=+0.029001792",
+	"Endpoints": [
+		"/v0.3/traces",
+		"/v0.3/services",
+		"/v0.4/traces",
+		"/v0.4/services",
+		"/v0.5/traces",
+		"/v0.5/stats",
+		"/profiling/v1/input"
+	],
+	"FeatureFlags": [
+		"feature_flag"
+	],
+	"Config": {
+		"Enabled": true,
```

Comment:
Upper case seems rather strange for JSON.  To me, camelCase would be preferable for Java.
But I guess there's no real agreement on this: https://stackoverflow.com/questions/5543490/json-naming-convention-snake-case-camelcase-or-pascalcase

Reply:
Yeah I'm with you. I also found this weird at firsts, but then slowly got used to it. It's a "Go thing", but it can be changed if needed, with some tedious work 🙂